### PR TITLE
Create unit test for making sure queries were called in the LateDayView controller

### PR DESCRIPTION
**Pull request by dakriy**
Also created a testing element to make this sort of testing trivial.

### CODE REVIEW

File: site/app/controllers/admin/LateController.php [hunk 1]
```diff
@@ -21,13 +21,12 @@ class LateController extends AbstractController {
...
      */
     public function viewLateDays() {
-        $students = $this->core->getQueries()->getAllUsers();
         return Response::WebOnlyResponse(
             new WebResponse(
                 ['admin', 'LateDay'],
                 'displayLateDays',
                 $this->core->getQueries()->getUsersWithLateDays(),
-                $students
+                $this->core->getQueries()->getAllUsers()
```
Comment (PlasmaIntec) [inline site/app/controllers/admin/LateController.php]: This change seems unrelated to the test itself

File: site/tests/app/controllers/admin/LateControllerTester.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        $this->core = $this->createMockCore([], [], [
+            'getUsersWithLateDays'  => [],
+            'getAllUsers'           => [],
+        ]);
+
+        $this->controller = new LateController($this->core);
+    }
+
+    /** @test */
+    public function it_tests_queries_being_made_in_view_late_days()
```
Comment (PlasmaIntec) [inline site/tests/app/controllers/admin/LateControllerTester.php]: Change function identifier to testQueriesMadeInLateDaysView to conform to namespace convention

Comment (dakriy) [reply site/tests/app/controllers/admin/LateControllerTester.php]: Thanks for the feedback! Fixed!
